crates/core/component/dex/src/component/position_manager/inventory_index.rs

```diff
@@ -0,0 +1,84 @@
+use cnidarium::StateWrite;
+
+use crate::{
+    lp::position::{self, Position},
+    state_key::eviction_queue,
+    DirectedTradingPair,
+};
+
+use anyhow::Result;
+use position::State::*;
+
+pub(super) trait PositionByInventoryIndex: StateWrite {
```

Comment:
Can we add docs to this index describing where it's used? This is the index used for the position eviction logic, but there's another index of _assets_ by inventory that's used for routing heuristics, and we should disambiguate between those.

Reply:
Agreed, though I'd prefer to document this with the eviction queue PR